component/mcsd/updater.go

```diff
@@ -119,6 +120,7 @@ func buildUpdateTransaction(tx *fhir.Bundle, entry fhir.BundleEntry, fhirBaseURL
 		requestMethod = entry.Request.Method
 	}
 
+	log.Info().Msgf("Updating resource %s", *entry.FullUrl)
```

Comment:
Think it was leftover from my debugging, will change to trace

```suggestion
	log.Ctx(ctx).Trace().Msgf("Updating resource %s", *entry.FullUrl)
```